model/delivery/LtiTestTakerAuthorizationService.php

```diff
@@ -112,4 +115,47 @@ public function isSuitable(User $user, $deliveryId = null)
         ], $user->getRoles());
         return !empty($ltiRoles);
     }
+
+    protected function isMissingProctorAuthorization(DeliveryExecutionInterface $deliveryExecution, User $user)
+    {
+        if ($this->isInitialStart($deliveryExecution) && $this->isAutostartEnabled($user)) {
+            return false;
+        }
+
+        return parent::isMissingProctorAuthorization($deliveryExecution, $user);
```

Comment:
Let's update `composer.json` with a projected version dependency on https://github.com/oat-sa/extension-tao-proctoring.

Reply:
Right, I've updated the dependency